ui/packages/catalog-ui-search/src/main/webapp/react-component/container/table-export/table-export.tsx

```diff
@@ -67,6 +67,23 @@ function getHiddenFields(): string[] {
     .get('columnHide')
 }
 
+function getHits(statuses: any[]): number {
```

Comment:
✏️ Since we have the benefits of types why not make it something like `statuses: Status[]` instead with

```
type Status = {
  id: string
  hits: number
}
```